plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/Fabric8WorkspaceEnvironmentProvider.java

```diff
@@ -115,17 +116,19 @@ public Config getWorkspacesOpenshiftConfig(Subject subject) throws Infrastructur
     }
 
     String userId = subject.getUserId();
+    String osoProxyUrl = multiClusterOpenShiftProxy.getUrl();
 
     if (cheServiceAccountTokenToggle.useCheServiceAccountToken(userId)) {
-      LOG.debug("Using Che SA token for '{}'", userId);
-      String osoProxyUrl = multiClusterOpenShiftProxy.getUrlWithIdentityIdQueryParameter(userId);
-      configBuilder.withMasterUrl(osoProxyUrl).withOauthToken(cheServiceAccountToken);
+      LOG.info("Using Che SA token for '{}'", userId);
+      config =
+          configBuilder.withMasterUrl(osoProxyUrl).withOauthToken(cheServiceAccountToken).build();
+      LOG.info("Adding Impersonate Header '{}'", userId);
```

Comment:
Do we need those 2 logs entries on an info level?

Reply:
just for a short period of time. Eventually, those log entries are going to be removed. For now those two on the `info` level are handy due to investigation of proxy issues like https://github.com/openshiftio/openshift.io/issues/3884